clients/geth/specular/rollup/utils/fmt/fmt.go

```diff
@@ -0,0 +1,42 @@
+package fmt
+
+import (
+	"fmt"
+	"io"
+)
+
+func Fprintf(w io.Writer, format string, a ...any) (n int, err error) {
+	return fmt.Fprintf(w,format,a...)
```

Comment:
nit: did you run gofmt? the spacing looks wrong

Reply:
formatting fixed